src/plugins/splitOperation.ts

```diff
@@ -0,0 +1,98 @@
+import { Host, Session } from "@azure-tools/autorest-extension-base";
+import { CodeModel, Request, Operation, Parameter } from "@azure-tools/codemodel";
+import { isNullOrUndefined } from "util";
+import { Helper } from "../helper";
+import { CliConst, CliCommonSchema } from "../schema";
+import { NodeHelper } from "../nodeHelper";
+import { Modifier } from "./modifier/modifier";
+import { CopyHelper } from "../copyHelper";
+
+export class SplitOperation{
+
+    constructor(protected session: Session<CodeModel>){
+    }
+
+    public async process() {
+
+        await this.modifier();
+
+        for (const group of this.session.model.operationGroups) {
+            const existedNames = new Set<string>(group.operations.map((op) => op.language.default.name.toUpperCase()));
```

Comment:
>default.name [](start = 90, length = 12)

better to add a comment about why default.name is used instead of cliKey or language->cli->name

Reply:
added comment. Thanks